### PR TITLE
React Redux styling

### PR DESCRIPTION
**Pull request by sainingo**
# React Styling
> In this branch I added a bit of styling to my react app.
1. Styled using the normal css

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -26,8 +26,7 @@ const Books = (props) => {
         <button type="button">Edit</button>
       </div>
 
-      <div>
-        <span>metric</span>
+      <div className="middle-data">
         <p>64%</p>
         <span>completed</span>
       </div>
```
Comment (ShoiraTa) [inline src/components/Books.js]: - [ ] You have done a great job, and the page looks awesome, but based on the requirements, your page should be EXACTLY like the wireframe. Currently I can see a number of differences. 

Your design  | Zeplin design
------------- | -------------
Screenshot 2022-05-12 at 13 11 01 | Screenshot 2022-05-12 at 13 09 49

More specifically, 

- [ ] Bookstore should be inside a container with shadow. 
- [ ] Please pay attention to background colors shadows and font colors and try to match them 
- [ ] Kindly use the same font
- [ ] Navbar's active link (Books) should be darker than than non active (categories)
- [ ] Kindly use dividing lines between `Comment` `Remove` and `Edit`
- [ ]  Kindly add dividing line between add `new book` and the table. 
- [ ] Kindly style the inputs accordingly
- [ ] Kindly add circular progress bar. You can use `CircularProgressbar` library to implement that:
```
import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';

            <div className="progress-div">
              <CircularProgressbar
                value={64}
                strokeWidth={8}
                styles={buildStyles({
                  strokeLinecap: 'butt',
                  pathTransitionDuration: 0.5,
                  pathColor: '#0290ff',
                  trailColor: '#DFE3E8',
                  backgroundColor: '#3e98c7',
                })}
              />
            </div>
            <div className="text-div">
              <p className="presentage-text">64%</p>
              <p className="completed-text">Completed</p>
            </div>
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @sainingo ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- You passed all the Linters! ✔️ 
- All the functionality works as expected! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+// import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
```
Comment (Kingstalux) [inline src/components/Books.js]: - [ ] (OPTIONAL) Kindly remove this piece of commented code to keep your work neat 👍

File: src/components/Books.js [hunk 3]
```diff
@@ -27,10 +30,12 @@ const Books = (props) => {
...
-        <span>completed</span>
+        <img src={progress} alt="progress" />
+        <div className="percent">
+          <p>64%</p>
+          <span>completed</span>
+        </div>
       </div>
-
       <div>
         <h4>CURRENT CHAPTERS</h4>
```
Comment (Kingstalux) [inline src/components/Books.js]: - Great job so far 👍 You've implemented most of the changes requested for the styling but there are a few which you have missed.
- [ ]  Kindly add the line dividing the `add new book` and the table.
- [ ] (OPTIONAL) Prove the correct font-weight for the `Bookstore cms` and `Add a new book` headings.
- [ ] Kindly use the correct font color for the `Current chapter` and the author of each book.
- [ ] Kindly use the correct `update progress` button color 👍 
- Your design:
Annotation 2022-05-12 124416
- Required design:
Annotation 2022-05-12 124404

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @sainingo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966


 ## Highlights 👏 
✅ Linters are passing 👍 
✅ Correct Gitflow used 👏 
✅ Good commit messages 🥂 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @sainingo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**